t/integration/test_tasks.py

```diff
@@ -27,7 +27,7 @@ class test_class_based_tasks:
     def test_class_based_task_retried(self, celery_session_app,
                                       celery_session_worker):
         task = ClassBasedAutoRetryTask()
-        celery_session_app.tasks.register(task)
+        celery_session_app.register_task(task)
```

Comment:
@thedrow Integration tests were not starting for the reason of celery_class_tasks. This fixture is used inside celery.contrib.pytest that is using it in celery_session_worker. The loop for class_task in celery_class_tasks was assigning task-based signature by celery_session_app.tasks.register that is outdated interface. Here is discussion that I found about that https://stackoverflow.com/questions/41788017/register-celery-class-based-task
	
Related issues: https://github.com/celery/celery/issues/5992